frontend/ui/size.lua

```diff
@@ -50,6 +50,7 @@ local Size = {
         small = Screen:scaleBySize(2),
         large = Screen:scaleBySize(10),
         button = Screen:scaleBySize(2),
+        buttontable = Screen:scaleBySize(4),
```

Comment:
Alternatively, increase button?

Reply:
There are so many things that are Buttons (that don't look like Buttons :) but just need to be tap'able - like our chevrons, the Dict lookup result word...) that would then be changed and made taller, resulting in taller everything :) that would really be noticable.
I don't remember having any tap issue with other things than these ButtonTable - so I'd rather not change regular Buttons.

Reply:
Well, besides the "real" buttons, doesn't that logically mean that the buttons that don't look like buttons are hard to tap too? :-)

I'm more immediately concerned about consistency with other buttons in general though.

Reply:
Logically, less hard (cause they got 2) than ButtonTables (who got 0).
But by experience, no issue.
"Other" buttons may also have a bigger font size, or some increase paddings.
We'll see how we feel about other Buttons now being less tall than ButtonTables/ButtonDialogs - but for now, I don't want to engage in a global UI entall'enment :)

Reply:
@NiLuJe Any opinion?

Reply:
I'm with @poire-z, the only potentially annoying to hit buttons are always on a button table, AFAIK.

Reply:
The only buttons I ever misclick (but instead accidentally dismiss a dialog) are *not* on a ButtonTable. :-P

Reply:
Just to be clear: buttons at the bottom of ConfirmBox are in a ButtonTable, and affected by my changes. They are all taller now (the only one bothering me a bit is the View HTML text viewer, with its View xxx.css, View yyy.css, Switch to debug view, Close, each alone on its own line - which takes a bit of vertical space out of the HTML text - but I'll be fine with it).

(The other "buttons" I can think of in a dialog are may be the `+1 -1 <<` in the Frontlight widget or the SkimTo widget , and they are quite far from the borders to cause a dismiss if missed :)